discovery-server/src/main/java/com/facebook/airlift/discovery/server/DiscoveryServer.java

```diff
@@ -53,7 +53,7 @@ public static void main(String[] args)
                     new DiscoveryModule()
             );
 
-            Injector injector = app.strictConfig().initialize();
+            Injector injector = app.initialize();
```

Comment:
curious: why remove `strictConfig` in all places?

Reply:
strictConfig is becoming the default in Airlift 0.198, and there is a method `noStrictConfig()` in case it needs to be disabled.